test/actions/prev.spec.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-unused-expressions */
+import { expect } from 'chai';
+import nock from 'nock';
+import { RSAA, apiMiddleware } from 'redux-api-middleware';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import rio, {
+  LOAD_REQUEST,
+  LOAD_SUCCESS,
+  LOAD_ERROR,
+  apiStateMiddleware,
+  JSON_API_SOURCE,
+  collection,
+} from '../../src';
+import { setStatus, updateStatus, getStatus } from '../../src/status';
+import { APPEND_MODE } from '../../src/actions/find';
+import { RESOLVED_ENDPOINT, NO_MORE_RESULTS } from '../../src/consts';
+import  { prev } from '../../src/actions/prev';
+
+describe('Next action creator', () => {
```

Comment:
Please re-evaluate tests and change everything into `prev`. Also run `npm run coverage` and open `coverage/index.html` in browser to see if everything is covered with tests.